Healthboard admissions od im

iainma09 opened the pull request:
Hi Calum, could you have a look over, thanks

iainma09 committed:
Need to tidy up edites to occupancy. Taking Calum's output, reformatting for Opendata, exporting and then importing to the "transfer_admissions_byhb_od.R" Still some formatting of the OpenData output but close to finalising

iainma09 committed:
create new ouput for Open Data use, needs to be on a weekly not daily basis
combine admissions and occupancy  files from output folder and join to a weekly framework which has a row for all health boards so that there are no missing gaps if no data supplied. output to open data

iainma09 committed:
create sour transfer script  in data_transfer.
Remove g_weekly_D occupancy from occupancy script

Calumm07 reviewed (changes requested):
Looks good and numbers match the dashboard. Just need to remove the line renaming the golden jubilee as that will be merged in with my branch and then it can be merged

iainma09 committed:
pop_transfer remove rm at this stage, can do it at end of data transfer
remove occupancy and admissions from admissions script
add NHS Tayside to admissions healthboard script (missed out in error)
Carehome remove TEMP output, not needed

iainma09 committed:
correct variable typo

iainma09 committed:
dashboard data transfer: tidy up the script add new source lines for separated out scripts e.g. carehome_time series and ethnicity. Add values from scripts so all in the one place.
admisssions removeweekly open data and occupancy section
create times_series_od and ethnicity_od script s
remove time series and ethnicity_od sections from originalscripts
tidy up rm section of occupancy to remove lare g_occupancy_hospital_hb
age_sex cases and simd cases: move od dates to data transfer
remove old weekly simd_od as not used

iainma09 commented:
Hi Calum, I've made changes to the data transfer process which are now ready for review.

I've created a new script for ethnicity open data as we don't always run ethnicity, but we need to bring the output version in, , reformat and export to the open data folder. 
Something similar too for the care_home_time series. the first part of the original  isn't working and there's a bit of confusion if we still need it. So I've just lifted the timeseries parts to a new script. 
I've removed the weekly admissions and occupancy section of the admissions script as we now have a new process. This has caused a conflict, but I think we just need to remove the whole section before merging to resolve it. I can chat through that with you if you like. 
I've removed the g_occupancy_hospital_hb at the end of the occupancy script just to tidy up this big df which I don't think is needed elsewhere.
Also done some tidying of  open_data names and values from some scripts and moved them into the data_transfer script

Calumm07 committed:
Merge branch 'new_structure_cm' into healthboard_admissions_od_im

Calumm07 reviewed (approved):
Produces all the files for me
